pkg/otlp/model/go.mod

```
@@ -12,7 +12,7 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/stretchr/testify v1.8.1
 	go.opentelemetry.io/collector/pdata v1.0.0-rc4
```

needs an update here too
```suggestion
	go.opentelemetry.io/collector/pdata v1.0.0-rc5
```